trigger-package-build.sh

```diff
@@ -82,74 +75,28 @@ main() {
             git add "$EXCLUDE_FILE"
```

Comment:
print-build-excludes.sh also depends on the git history. You will need to use the GitHub actions query API for this or something.

Reply:
Got it, I would rewrite `print-build-excludes.sh`.

Reply:
It seems that `gh run` does not give me any information about the `inputs` provided to the workflow.

I'm now thinking of directly creates a commit to the excludes.txt on failure of `bulid-package`.

Reply:
I've implemented a new mechanism for generating excludes, however since none of the build has failed yet, it's not tested.